templates/webapi/main/test_result_overview_link.html.ep

```diff
@@ -0,0 +1,5 @@
+<a href="<%= url_for('tests_overview')->query(todo => 1, map { (groupid => $_->{id}) } @$group_list); %>"
```

Comment:
I just realized now on OSD in production that this query includes "todo=1" which was unexpected. The link says "test overview", not "all tests needing review" so please delete that query parameter part in its own commit please.

Reply:
Done

Reply:
Please in a separate commit as you effectively fix the problem that is apparent in the already merged and deployed state

Reply:
Now handled in https://github.com/os-autoinst/openQA/pull/4013 explicitly to be applied as fix first.

Reply:
Rebased from Master